src/conversion_helper.js

```diff
@@ -501,6 +507,17 @@ function roundToDecimalPlaces(number, places) {
   return (Math.round(number * multiplier)/multiplier).toFixed(places);
 }
 
+function ContainsArray(wrapper, main) {
```

Comment:
Let me know if I an describing this function correctly: Given an array "wrapper" and a  number "main",  this function returns "true" if "main" JSON.stringify-equals any item in array "wrapper". Is that correct?

Why do we need to JSON.stringify an item to match it?

Also, I feel like this might be clearer:
- if we rename the variables to be more descriptive of what they represent. i.e. "array" and "target" instead of "wrapper" and "main"
- if we used a regular "for" or "for-in" loop, instead of a forEach with an external "ret" value

Reply:
> Why do we need to JSON.stringify an item to match it?

Since we are comparing arrays and javascript doesn't allow direct comparison of arrays. There are other solution, but I find this more straightforward.

I'll make the other changes.

Reply:
Oh, I misunderstood the inputs. I thought that 'main' was a number, I didn't realize it was an array. Maybe renaming it to "targetArray" would clear up the confusion? I think stringifying it is fine, in this case :)

Reply:
yep, I'll change `number` to `numbers` later.